src/inspect_ai/_view/server.py

```diff
@@ -275,13 +299,21 @@ def normalize_uri(uri: str) -> str:
 
 
 def log_listing_response(logs: list[EvalLogInfo], log_dir: str) -> web.Response:
+    # Cleanly handle path names, TODO: add support for other cloud providers with multiple access points
+    def normalize_name(name: str) -> str:
```

Comment:
Let's make this a helper function alongside the other azure-specific stuff you factor out from above. Generally I think we should have a `_view/azure.py` source file where all of this lives.